_posts/2022-01-10-reassign-partitions.md

```diff
@@ -42,17 +34,38 @@ Some of these are listed here:
 1. You can reassign the partition between the brokers anytime. For example, it can be used at times when you want to scale down the number of brokers in your cluster.
    You can assign partitions from the broker to be scaled down to other brokers which will handle these partitions now.
 
-2. With the help of this tool, you can increase the no. of partitions / replicas which can help with increasing the throughput of the topic.
+2. With the help of this tool, you can increase the number of partitions / replicas which can help with increasing the throughput of the topic.
+
+## Partition Reassignment Throttle
+Partition reassignment tool reassigns the partition between the brokers and sometimes this process could be slow due to transfer of large data.
```

Comment:
`Partition reassignment tool` -> `The partition reassignment tool`
`reassigns the partition` -> `reassigns the partitions`.

What about something like:

> 
```suggestion
Reassigning partitions between brokers can sometimes lead to a large transfer of large data. To avoid overloading the cluster, it is recommended to always set a throttle rate to limit the bandwidth used by the reassignment. This can be done using the `--throttle` flag which sets the maximum allowed bandwidth in bytes, for example `--throttle 5000000` sets the limit to 50 MB/s.
```